parser/misc.go

```diff
@@ -119,6 +119,8 @@ func init() {
 	initTokenString("<>", neqSynonym)
 	initTokenString("<<", lsh)
 	initTokenString(">>", rsh)
+	initTokenString("->", jss)   // jss means json separator symbol.
```

Comment:
Those two line is useless, as the exists of `initTokenFunc("-", startWithDash)`, remove them.